src/cmd/commit.ts

```diff
@@ -154,7 +154,20 @@ export const commit = async (): Promise<void> => {
   if (values?.charts?.gitea?.enabled) await commitAndPush()
   else d.log('The files have been prepared, but you have to commit and push to the remote yourself.')
 
-  if (isChart) await setDeploymentStatus()
+  if (isChart) {
+    await setDeploymentStatus()
+    const credentails = values!.charts.keycloak
```

Comment:
if the word "credentails" is not caught with spellcheck, then is that not running in your environment?

Reply:
done